nx-dev/feature-search/src/lib/algolia-search.tsx

```diff
@@ -140,8 +140,8 @@ export function AlgoliaSearch({
             placeholder="Search the docs"
             initialScrollY={window.scrollY}
             onClose={handleClose}
-            indexName="nx-production"
-            apiKey="f49a1eb671385f0472a7285556168930"
+            indexName="nx-canary-production"
+            apiKey="07340b85db83fd49c1f904f883cc3ef8"
```

Comment:
We're replacing the production index with the canary index?  What about people that are on the main nx.dev site?

Reply:
https://nx.dev will point to whatever version is current and its index will be updated once it starts to point at a version other than master.